app/controllers/tt/api/v1/visitor_details_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Tt::Api::V1::VisitorDetailsController < Tt::Api::V1::ApplicationController
+  def show
+    render json: TestTrack::FakeServer.visitor_details
```

Comment:
Yea, the idea is that these things could become stateful over time, if we find the need. So, centralizing it is nice. Also, FakeServer has a seed fed into it so it's good that we have them all hanging off of the central object so that we can easily leverage the seed or other injected bits if we so desire.